src/components/RadioButton/RadioButton.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import "./RadioButton.scss";
+
+export default function RadioButton({
+  disabled = false,
+  label = "Default checkbox",
+  checked = false,
+  handleClick = () => {},
+  ...props
+}) {
+  return (
+    <div className="form-check">
+      <input
+        className="form-check-input"
+        type="checkbox"
+        value=""
+        checked={checked}
+        id="flexCheckDefault"
+        disabled={disabled}
+        onClick={handleClick}
+        {...props}
+      />
+      <label className="form-check-label" htmlFor="flexCheckDefault">
+        {label}
+      </label>
+    </div>
+  );
+}
```

Comment:
All code is great, just the filename is RadioButton.js and should be CheckboxButton.js